config/locales/fil/restroom.fil.yml

```diff
@@ -0,0 +1,64 @@
+fil:
+  activerecord:
+    attributes:
+      restroom:
+        # only overridden attributes here, defaults work as expected
+        # (e.g. database field 'city' will be appear as 'City')
+        name: 'Lokasyon/Lugar/Pangalan ng Negosyo'
+        street: 'Ang Street Address'
+        state: 'Estado/Probinsya'
+        access: 'Madaling ma-akses?'
+        bath_type: 'Uri ng Pahingaan o Banyo'
+
+  search_bar:
+    enter_location: 'I-enter ang address'
+
+  restroom:
+    add_new: 'Magsumita ng pahingaan o banyo sa aming database'
+    preview: 'Prebyu'
+    required: '* markahan ang kailangan na field'
+    guess_location: 'Magpalagay ng kasalukuyang lokasyon'
+    directions: 'Direksyon'
+    directions_hint: '(hal.: ikatlong palapag sa likod, sa may bihisanang kwarto...)'
+    distance:
+      miles: 'milya'
+      kilometers: 'kilometro'
+    comments_hint: '(hal.: ikaw ay dapat "paying na kustomer", ikaw ay dapat na kumilos na parang nag-browsing para sa bit...)'
```

Comment:
> Also, I can't find "kustomer" in online dictionaries, so if you would be so kind, can you confirm this is a word?

I'm not sure if you can find this in online dictionaries since it likes a translation just by changing of letter. This is like the same translation of "Navigation" => "Nabigasyon"